app/products/calls/components/current_call_bar/current_call_bar.tsx

```diff
@@ -143,6 +146,17 @@ const CurrentCallBar = ({
 
     const micPermissionsError = !micPermissionsGranted && !currentCall?.micPermissionsErrorDismissed;
 
+    // The user should receive an alert if all of the following conditions apply:
+    // - Recording has started.
+    // - Recording has not ended.
+    // - The alert has not been dismissed already.
+    if (currentCall?.recState?.start_at && !currentCall?.recState?.end_at && !currentCall?.recAcknowledged) {
+        if (!recordingAlertExists) {
+            recordingAlert(intl);
+            setRecordingAlertExists(true);
```

Comment:
This guards against the call screen throwing up two alerts right after each other.